examples/notebooks/Absorption_profile_AlGaAs_GaAs_structure.ipynb

```diff
@@ -0,0 +1,198 @@
+        "# Absorption Profile AlGaAs GaAs Structure Example"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "9282DOR42thA",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "import matplotlib.pyplot as plt\n",
+        "import numpy as np\n",
+        "from solcore.absorption_calculator import calculate_absorption_profile\n",
+        "from solcore import material, si\n",
+        "from solcore.structure import Structure, Layer"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "U9-mFKNO22IZ",
+        "colab_type": "text"
+      },
+      "source": [
+        "First we defined a couple of materials, for example, GaAs and AlGAs"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "dHoZMpKB25ZY",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "GaAs = material('GaAs')(T=300)\n",
+        "AlGaAs = material('AlGaAs')(T=300, Al=0.3)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "qmA8l1ja272Y",
+        "colab_type": "text"
+      },
+      "source": [
+        "Now, let's build the structure"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "d5DpfgwH2-rp",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "my_structure = Structure([\n",
+        "    Layer(si(30, 'nm'), material=AlGaAs),\n",
+        "    Layer(si(3000, 'nm'), material=GaAs),\n",
+        "    Layer(si(300, 'um'), material=GaAs),\n",
+        "])"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "q-p7bGCM3BAJ",
+        "colab_type": "text"
+      },
+      "source": [
+        "We want to calculate the absorption profile of this structure as a function of the position and wavelength"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "ihObNLeu3C4p",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "wl = np.linspace(400, 1000, 200)\n",
+        "out = calculate_absorption_profile(my_structure, wl, steps_size=1, z_limit=3000)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "rq_rk3S13Enp",
+        "colab_type": "text"
+      },
+      "source": [
+        "Finally, we plot the absorption profile. Note that absorption at short wavelengths take place near \n",
+        "\n",
+        "the surface of the structure, in the AlGaAs layer and top of the GaAs layer, while longer wavelengths\n",
+        "\n",
+        "penetrate more. Wavelengths beyond the GaAs band edge are not absorbed."
```

Comment:
This three lines are actually a paragraph, but as they have been copy/paste from the original file, they appear as separate independent lines in the Notebook. It will be great if you could remove the unnecessary end-of-line, so they do appear as a paragraph.